Makefile

```diff
@@ -39,8 +39,8 @@ endif
 
 # Auto-version the source code and create a GitHub release
 version:
-	@if [ "$(REGION)" == "QA" ] || [ -n "$(CI)" ]; then \
-		if [ -z "$(SOURCE_VERSION)" ]; then echo "Error: SOURCE_VERSION is not set" && exit 0; fi; \
+	@if [ "$(REGION)" = "QA" ] || [ -n "$(CI)" ]; then \
```

Comment:
this giant shell-if-embedded-in-make scares me

Reply:
It scares me too, I mostly got lazy and couldn't work out a nice way to do `or` with Make conditions. I'm 90% sure I'll refactor when I move it into our shared Makefile

Reply:
you _can_, but it's nasty. you concatenate the all the things you're comparing. as in concatenate the strings and compare those. and thats an or. don't even go there